demo/Dockerfile_snapshot

```diff
@@ -48,8 +48,10 @@ RUN /usr/local/bin/gen-security-data.sh /tmp/files/regup/sec
 COPY run-demo.sh /usr/local/bin/run-demo.sh
 
 COPY plugins.txt /tmp/files/
+RUN sed -i "s/docker-workflow:@VERSION@//g" /tmp/files/plugins.txt
```

Comment:
Nope, I am just deleting the entire line; `docker-workflow.jpi` is copied manually (`ADD snapshot-plugins …`).